src/components/Layout/Layout.stories.tsx

```diff
@@ -165,20 +156,23 @@ function TestLayout({ children }: PropsWithChildren<{}>) {
   return (
     <PreventBrowserScroll>
       <TestTopNav />
-      <ScrollableParent>{children}</ScrollableParent>
+      <ScrollableParent paddingX={increment(3)}>{children}</ScrollableParent>
     </PreventBrowserScroll>
   );
 }
 
 function TestProjectLayout({ children }: PropsWithChildren<{}>) {
   return (
-    <TestLayout>
+    <PreventBrowserScroll>
+      <TestTopNav />
       {/* Required to use `overflowHidden` as the prevent the `TestLayout`'s scrollbar from kicking in. */}
       <div css={Css.df.overflowHidden.$}>
         <TestSideNav />
-        <ScrollableParent xss={Css.fg1.$}>{children}</ScrollableParent>
+        <ScrollableParent xss={Css.fg1.$} paddingX={increment(3)}>
+          {children}
+        </ScrollableParent>
       </div>
-    </TestLayout>
+    </PreventBrowserScroll>
```

Comment:
One slight bummer about supplying the padding on the ScrollableParent component is that nesting ScrollableParents could lead to doubling up the padding. This would mean that we can't really nest Layouts.